srcs/semaphore.cc

```diff
@@ -139,59 +132,32 @@ void Semaphore::Acquire(const Nan::FunctionCallbackInfo<v8::Value>& info)
     return ;
   }
   obj->locked = true;
-  if (obj->debug) {
-    int sval;
-    if (sem_getvalue(obj->semaphore, &sval)) sval = -1;
-    printf("[posix-semaphore] After sem_wait, value = %d\n", sval);
-  }
 }
 
 void Semaphore::Release(const Nan::FunctionCallbackInfo<v8::Value>& info)
 {
   Semaphore* obj = ObjectWrap::Unwrap<Semaphore>(info.Holder());
-  
-  /*if (!obj->strict && (obj->closed || !obj->locked))
-  {
-    if (obj->debug)
-      printf("[posix-semaphore] 'release' called when semaphore was already released or closed, but strict mode deactivated, so not failing\n");
-    return ;
-  }*/
+
   if (obj->strict && obj->closed)
     return Nan::ThrowError("trying to do operation over semaphore, but already closed");
   if (obj->strict && !obj->locked)
     return Nan::ThrowError("trying to release semaphore, but already released");
-  if (obj->debug) {
-    int sval;
-    if (sem_getvalue(obj->semaphore, &sval)) sval = -1;
-    printf("[posix-semaphore] Before sem_post, value = %d\n", sval);
-  }
   if (sem_post(obj->semaphore) == -1)
   {
     if (obj->debug || !obj->silent)
     {
-      printf("[posix-semaphore] sem_post failed, printing errno message ('man sem_post' for more details on possible errors) : \n");
+      printf("[posix-semaphore] sem_post failed, printing errno message ('man sem_post' for more details on possible errors): \n");
       perror("[posix-semaphore] ");
     }
     return Nan::ThrowError("could not release semaphore, sem_post failed");
   }
   obj->locked = false;
-  if (obj->debug) {
-    int sval;
-    if (sem_getvalue(obj->semaphore, &sval)) sval = -1;
-    printf("[posix-semaphore] After sem_post, value = %d\n", sval);
-  }
 }
 
 void Semaphore::Close(const Nan::FunctionCallbackInfo<v8::Value>& info)
 {
   Semaphore* obj = ObjectWrap::Unwrap<Semaphore>(info.Holder());
 
-  /*if (!obj->strict && obj->closed)
```

Comment:
Is there a reason to remove this block? Maybe it was left out after you were done debugging?

Reply:
Ah sorry, failed to see that it was commented out.